src/main/java/de/komoot/photon/utils/ConvertToJson.java

```diff
@@ -0,0 +1,96 @@
+package de.komoot.photon.utils;
+
+import com.google.common.collect.Lists;
+import de.komoot.photon.Constants;
+import lombok.extern.slf4j.Slf4j;
+import org.elasticsearch.action.search.SearchResponse;
+import org.elasticsearch.search.SearchHit;
+import org.json.JSONArray;
+import org.json.JSONObject;
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Convert a elasticsearch {@link SearchResponse} into a list of {@link JSONObject}s
+ * <p/>
+ * Created by Sachin Dole on 2/20/2015.
+ */
+@Slf4j
+public class ConvertToJson implements OneWayConverter<SearchResponse, List<JSONObject>> {
```

Comment:
i did not get this one: what's the difference between ConvertToJson and ConvertGeoToJson?